lib/topology/platforms/base.py

```diff
@@ -439,5 +439,20 @@ def disable(self):
         """
         self._enabled = False
 
+    def _log_command(self, command):
+        """
+        Command logging function for low-level shell API usage.
+
+        FIXME: how can we specify the shell here?
```

Comment:
@carlos-jenkins, any suggestions on how to pass the used shell name from the node namespace to the shell namespace?